modules/templates-rails-creating-frontend-service.adoc

```diff
@@ -73,12 +78,14 @@ by running the database migration script. There are two ways you can do this:
 +
 ** Exec into frontend container with `rsh` command:
 +
+[source,terminal]
 ----
   $ oc rsh <FRONTEND_POD_ID>
```

Comment:
There is some indentation that should be removed here, and can you change the replaceable value from <FRONTEND_POD_ID> to <frontend_pod_id> please?

The below code block is indented too.